common/src/main/java/com/wynntils/core/rendering/objects/RenderPosition.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.rendering.objects;
+
+import com.mojang.blaze3d.platform.Window;
+import com.wynntils.mc.utils.McUtils;
+
+public class RenderPosition {
+    /** Position along the x and y-axis as a percentage of the total screen width */
+    public float x;
+
+    public float y;
+
+    public int width;
+    public int height;
+
+    /** A cached value for the x and y-axis position according to the current screen size */
+    private float drawingX;
+
+    private float drawingY;
```

Comment:
```suggestion
    public float x;
    public float y;

    public int width;
    public int height;

    /** A cached value for the x and y-axis position according to the current screen size */
    private float drawingX;
    private float drawingY;
```

Reply:
Also please make all fields private and use getters.

Reply:
The spotless apply is the one that added those spaces, let me see if it would fail the check if I reversed those